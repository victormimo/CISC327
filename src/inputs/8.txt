login
agent
A
4670204
120000
B
4670204
120000
C
4670204
3543064
120000
logout
