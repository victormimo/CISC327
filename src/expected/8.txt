DEP 4670204 120000.0 (none) XDIRS
WDR 4670204 120000.0 (none) XDIRS
XFR 3543064 120000.0 4670204 XDIRS
EOS
